### PR TITLE
Force Menu + L4D2 Golf CLub

### PR DESCRIPTION
**Pull request by CombineSlayer24**

### CODE REVIEW

File: lua/lambdaplayers/lambda/weapons/l4d2_golfclub.lua [hunk 1]
```diff
@@ -0,0 +1,50 @@
+local random = math.random
+local CurTime = CurTime
+local Rand = math.Rand
+
+table.Merge( _LAMBDAPLAYERSWEAPONS, {
+
+    meathook = {
```
Comment (IcyStarFrost) [inline lua/lambdaplayers/lambda/weapons/l4d2_golfclub.lua]: This should be renamed

Comment (CombineSlayer24) [reply lua/lambdaplayers/lambda/weapons/l4d2_golfclub.lua]: Renamed

Comment (CombineSlayer24) [reply lua/lambdaplayers/lambda/weapons/l4d2_golfclub.lua]: Sorry if this is PR is messy. I never done a Pull request before.

**Review by CombineSlayer24 — COMMENTED** [review]
🤦, when you forget to remove something and you didnt catch it

**Pull request merged**